custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -217,6 +220,13 @@ def updates(self):
                 CASE WHEN %(month_end_11yr)s > dob AND %(month_start_15yr)s <= dob AND sex = 'F'
                 THEN 1 ELSE 0 END
             ) as cases_person_adolescent_girls_11_14_all,
+            SUM(CASE WHEN ( (out_of_school or re_out_of_school) AND
+                                 (not admitted_in_school )AND  migration_status<>1) THEN 1 ELSE 0 END ) as girls_out_of_schoool,
```

Comment:
I do not think this is correct logic. I think you need `migration_status IS DISTINCT FROM 1` otherwise, when `migration_status` is null (which is very common) this will return false

Reply:
good catch. changing now.

Reply:
Hey, But Wait. this is in a case when statement and null will be evaluated as false. so that will still be logically right no? @calellowitz